superset-frontend/src/components/EmptyState/index.tsx

```diff
@@ -31,7 +31,7 @@ export enum EmptyStateSize {
 export interface EmptyStateSmallProps {
   title: ReactNode;
   description?: ReactNode;
-  image: ReactNode;
+  image?: ReactNode;
```

Comment:
what necessitates this being optional?

Reply:
@AAfghahi There is an error message screen that re-uses this component but does not have a placeholder image.  The image optionality was added to make re-using the component more flexible.